scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -127,6 +127,7 @@
     "\n",
```

Comment:
<p>The code before was much simpler. Can we get this to a single (or maybe two) lines of code perhaps again.</p><p>eval_metrics = tracker.evaluate(eval_results, EVAL_DATA_PATH) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
didn't change evaluate(). Rewrite it by re-using sub-functions inside of evaluate().